Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -0,0 +1,117 @@
+# How to Configure Leap Motion Hand Tracking in MRTK 
+
+A [Leap Motion Controller](https://www.ultraleap.com/product/leap-motion-controller/) is required to use this data provider.
+
+The Leap Motion Data Provider enables articulated hand tracking for VR and could be useful for rapid prototyping in the editor.  The data provider can be configured to use the leap motion controller mounted on a headset or placed on a desk face up.
+
+## Using Leap Motion tracking in MRTK
+1. Prepare MRTK project for Leap Motion
+
+    - This step only applies if the source of MRTK is cloned from the git repo and NOT the unity packages.
+
+        - Navigate to **Mixed Reality Toolkit > Utilities > Update > Configure CSC File for Leap Motion**. Updating the csc file filters out the obsolete warnings produced by the Leap Motion Core Assets.
```

Comment:
For the menu paths, it feels a bit weird to me seeing leap configuration happen in two different places:

Mixed Reality Toolkit > Utilities > Update > Configure CSC File for Leap Motion
Mixed Reality Toolkit** > **Utilities** > **Leap Motion** > **Reconcile Leap Motion Define

It feels like maybe they should be in the same leaf folder

Reply:
If not, it would be cool to hear the rationale for why not

Reply:
The plan was to combine the two menu items into one, after the menu mentioned under TODO in the PR description was complete.